functests/performance/rt-kernel.go

```diff
@@ -59,4 +60,19 @@ var _ = Describe("[performance]RT Kernel", func() {
 
 	})
 
+	It("[test_id:28526][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Non worker-cnf node should not have RT kernel installed", func() {
+
+		By("Skipping test if cluster does not have another available worker node")
+		nonRTWorkerNode, err := nodes.GetNonCNFNode(testclient.Client, "worker")
+		Expect(err).ToNot(HaveOccurred())
+
+		if nonRTWorkerNode == nil {
```

Comment:
IIUC in this line you are verifying if the list of non-cnf nodes is empty? I think you need to use this:
if len(nonRTWorkerNode) == 0

Reply:
Done.